### PR TITLE
chore: add new errors

### PR DESCRIPTION
**Pull request by thevaibhav-dixit**

### CODE REVIEW

File: src/payout_queue/repo.rs [hunk 1]
```diff
@@ -46,21 +46,21 @@ impl PayoutQueues {
...
               ORDER BY e.sequence"#,
-            Uuid::from(account_id),
+            account_id as AccountId,
             name
         )
         .fetch_all(&self.pool)
         .await?;
         if rows.is_empty() {
-            return Err(BriaError::PayoutQueueNotFound(name));
+            return Err(PayoutQueueError::PayoutQueueNotFound(name));
```
Comment (bodymindarts) [inline src/payout_queue/repo.rs:63]: This needs to translate into status::not_found in src/api/server/convert.rs

File: src/app/mod.rs [hunk 2]
```diff
@@ -607,8 +607,10 @@ impl App {
     }
 
     #[instrument(name = "app.list_wallets", skip_all, err)]
-    pub async fn list_wallets(&self, profile: Profile) -> Result<Vec<Wallet>, BriaError> {
-        self.wallets.list_by_account_id(profile.account_id).await
+    pub async fn list_wallets(&self, profile: Profile) -> Result<Vec<Wallet>, ApplicationError> {
+        println!("{:?}", profile.account_id);
```
Comment (bodymindarts) [inline src/app/mod.rs:611]: ??

File: src/app/mod.rs [hunk 3]
```diff
@@ -607,8 +607,10 @@ impl App {
...
+        let wallets = self.wallets.list_by_account_id(profile.account_id).await?;
+        Ok(wallets)
```
Comment (bodymindarts) [inline src/app/mod.rs:613]: Why not return directly? The repo function type should line up

File: src/app/mod.rs (hunk 2 above)
Comment (thevaibhav-dixit) [reply src/app/mod.rs:611]: I was trying to test something there. Sorry. I will remove it.

**Comment by bodymindarts** [comment]
Closes https://github.com/GaloyMoney/bria/issues/208

**Pull request merged**